src/shared/interfaces/styles.interface.ts

```diff
@@ -1,11 +1,61 @@
 /**
- * Used by colors
+ * Used by Colors
  */
+interface Elevation {
+  "50": string;
+  "100": string;
+  "200": string;
+  "300": string;
+  "400": string;
+  "500": string;
+  "600": string;
+  "700": string;
+  "800": string;
+  "900": string;
+  a100?: string;
+  a200?: string;
+  a400?: string;
+  a700?: string;
+}
+
 export interface AppColors {
-  primary: string;
-  secondary: string;
-  success: string;
-  dark: string;
-  primaryTextColor: string;
-  secondaryTextColor: string;
```

Comment:
We should use an interface like this for all the components, since some users have their own colors charts, they will just update the config object with their own, instead of changing on every component.
Keep the color tree on the side to pick the colors of the default setup. We will see if we remove it later.

Reply:
Done